pkg/agent/controller/agent.go

```diff
@@ -78,23 +78,31 @@ func (a *Controller) Run(stopCh <-chan struct{}) error {
 	return nil
 }
 
-func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) runtime.Object {
+func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) (runtime.Object, bool) {
 	svcExport := obj.(*lighthousev2a1.ServiceExport)
 	svc, err := a.kubeClientSet.CoreV1().Services(svcExport.Namespace).Get(svcExport.Name, metav1.GetOptions{})
 	if err != nil {
 		klog.Errorf("No matching service for %v", svcExport)
-		return nil
+		return nil, true
```

Comment:
Need to handle delete use case here. Should wait till we get submariner-io/admiral#83 merged.

Reply:
It is merged but I think we should leave such changes for https://github.com/submariner-io/lighthouse/pull/188.  The purpose of this PR is only to switch to using `ServiceImport`. In fact I don't this PR should bump the admiral dependency to pick up the transform function signature changes - those are intended for  https://github.com/submariner-io/lighthouse/pull/188.

Reply:
Removed the changes.